.github/workflows/config/spelling_allowlist.txt

```diff
@@ -190,43 +198,59 @@ instantiations
 intrinsics
 iterable
 iteratively
+kets
 linker
 linkers
+logarithmically
 lookup
 lookups
 lossy
 lvalue
 makefiles
 merchantability
+mgpu
+mqpu
+multithreaded
 mutex
 namespace
 namespaces
 natively
+normalisation
+nvidia
 nullary
 optimizer
 optimizers
+parallelize
 parallelization
 parameterization
+pauli
 performant
+photonic
 precompute
 precomputed
 prepend
 preprocessor
+probabilites
 programmatically
 pybind
+qpp
 quantize
 quantized
 qubit
 qubits
 qudit
 qudits
+QPP
```

Comment:
Please guard the code reference in the docs/comments with backticks instead of adding them here (`qpp`)